pkg/acs/calacs/include/acs.h

```diff
@@ -13,6 +13,9 @@
 # define ACS_FITS_REC       82
 # define SZ_STRKWVAL        68
 
+#define ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN 4144
```

Comment:
Theoretically, yes. ACS does not have crazy cross-chip subarrays that I heard WFC3 has. Let's merge this and see if tests for subarrays pass tomorrow, just to be sure.